pkgs/tools/networking/smokeping/default.nix

```diff
@@ -0,0 +1,57 @@
+{stdenv, fetchurl, perl, buildPerlPackage, fping, rrdtool, perlPackages }:
+let
+
+  ConfigGrammar = buildPerlPackage {
```

Comment:
I think this (and rest of the perl packages) should go in pkgs/top-level/perl-packages.nix